modules/admin_manual/pages/configuration/server/security/hsmdaemon.adoc

```diff
@@ -0,0 +1,595 @@
+= The HSM (Hardware Security Module) Daemon (hsmdaemon)
+:description: Learn how to install ownCloud’s custom HSM (Hardware Security Module) and configure ownCloud to delegate encryption to it.
+:keywords: hsmdaemon, hardware security module, security, pkcs11, softhsm2
+:base64-encoding-url: https://en.wikipedia.org/wiki/Base64
+:hsm-url: https://en.wikipedia.org/wiki/Hardware_security_module
+:jwt-url: https://jwt.io/
+:network-sockets-url: https://en.wikipedia.org/wiki/Network_socket
+:opensc-wiki-url: https://github.com/OpenSC/OpenSC/wiki
+:opensuse-security-repositories-url: https://download.opensuse.org/repositories/security/
+:php-exec-function-url: https://www.php.net/manual/en/function.exec.php
+:pkcs11-url: https://en.wikipedia.org/wiki/PKCS_11
+:pkcs11-tool-url: https://linux.die.net/man/1/pkcs11-tool 
+:softhsm2-url: https://www.opendnssec.org/softhsm/
+:unix-sockets-url: http://beej.us/guide/bgipc/html/multi/unixsock.html
+
+The hsmdaemon is a daemon to delegate encryption to an {hsm-url}[HSM (Hardware Security Module)].
```

Comment:
It is just a small thing.
On the installation of the daemon below, it is first time mentioned that this daemon is provided by owncloud.
I would propose therefore to write here at the beginning:
`The hsmdaemon is a daemon to` --> `The hsmdaemon is a daemon, provided by ownCloud, to`.
Additionally I propose to add a note here in this top section how to obtain it, who is eligable to obtain it, as this document is NOT in the enterprise section.

Reply:
As always, well spotted.

Reply:
@sharidas or @patrickjahns, can you provide further details on how the daemon is obtained and who is able to receive a copy?

Reply:
This is something for @hodyroff and @pmaier1 ( or even @CaptionF )

Reply:
> @sharidas or @patrickjahns, can you provide further details on how the daemon is obtained and who is able to receive a copy?

So, there are two components to this:
- The `encryption` app is part of ownCloud Server and will have out-of-the-box support for the `hsmdaemon` with the upcoming 10.2 release.
- The `hsmdaemon` itself will then be available via our customer portal as part of the Enterprise Edition.